input_templates/terra_workspaces/single_sample/single_sample_workspace_dashboard.md.tmpl

```diff
@@ -0,0 +1,153 @@
+## GATK-SV Best Practices for Structural Variation Discovery on Single Samples
```

Comment:
I was wondering why this was a template? Was it just so that the README ends up in the same directory that's generated for the other Terra files?

Reply:
Yes, I just wanted all the files to create the workspace to be together in the same output directory for easy creation of the workspace.